### PR TITLE
add pandas-vet recipe

### PR DESCRIPTION
**Pull request by deppen8**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pandas-vet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - attrs
+    - flake8 >3.0.0
```
Comment (CJ-Wright) [inline recipes/pandas-vet/meta.yaml]: Is this a host dep? (also attrs)

Comment (deppen8) [reply recipes/pandas-vet/meta.yaml]: I am new to conda-forge and used `conda skeleton` to build the `meta.yml`, so I am not totally sure...but this package is a linter plugin for flake8, so it does need that to function. flake8 is not just for development of the package.

Comment (CJ-Wright) [reply recipes/pandas-vet/meta.yaml]: The host section is for packages which must be installed during the building. Most python packages only have a host dependence on python and pip.

Comment (deppen8) [reply recipes/pandas-vet/meta.yaml]: Thanks for catching this and pointing me in the right direction, @CJ-Wright . Looks like it builds successfully with the extraneous packages removed. Anything else I need to clean up?

**Comment by dbast** [comment]
Can this recipe be `noarch: python` with python >3.5 as host/run dependencies instead of the `skip`? See also https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python

**Comment by deppen8** [comment]
I think it could be `noarch; python`, but the package uses entrypoints, so I could use some guidance on specifying that correctly in `meta.yml`.

From `setup.py`:
```python
entry_points={
       'flake8.extension': [
            'PD = pandas_vet:VetPlugin',
        ],
    }
```

Would that translate to this?
```yml
build:
  entry_points:
    - PD = pandas_vet:VetPlugin
```

**Comment by chrisburr** [comment]
@deppen8 I'm fairly sure `entry_points` are only needed for `console_scripts` as it's the executables that need to be made platform specific. If so, you shouldn't need anything in this case.

File: recipes/pandas-vet/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - pandas_vet
+  requires:
+    - coveralls >=1.5.1
+    - pytest >4.0.0
+    - pytest-cov >=2.6.1
+
+about:
+  home: https://github.com/deppen8/pandas-vet
+  license: MIT
+  license_family: MIT
```
Comment (chrisburr) [inline recipes/pandas-vet/meta.yaml]: The MIT licence requires a copy of the text to be included in all distributions of the package. Ideally it should be included in the tarball but as it's missing you can also include a copy in the recipe directory.
```suggestion
  license_family: MIT
  licence_file: LICENCE
```

File: recipes/pandas-vet/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    - attrs
+    - flake8 >3.0.0
+    - python >=3.6
+    - setuptools
+
+test:
+  imports:
+    - pandas_vet
+  requires:
```
Comment (chrisburr) [inline recipes/pandas-vet/meta.yaml]: As this is only doing an import test I suspect these dependencies aren't needed? If they are required they should probably be runtime dependencies.

**Comment by deppen8** [comment]
Many thanks to @chrisburr, @CJ-Wright, and @dbast for helping a noob like me through this process!

**Pull request merged**